jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -183,6 +210,24 @@ String getBaseImageCredentialHelperName() {
     return Preconditions.checkNotNull(from).credHelper;
   }
 
+  /**
+   * Gets an {@code Authorization} from system properties, or from the {@code <from><auth>}
+   * configuration if the system properties are not set, or {@code null} if neither are set.
+   *
+   * @return a new {@code Authorization} from the configured username and password, or {@code null}
+   *     if one isn't configured.
+   */
+  @Nullable
+  Authorization getBaseImageAuth() {
+    // System property takes priority over build configuration
+    String commandlineUsername = System.getProperty("jib.from.auth.username");
```

Comment:
Can consolidate this logic with the `getTargetImageAuth` under a private static method.